### PR TITLE
OMG-443 exit id for deposit is not unique

### PR DESCRIPTION
**Pull request by pik694**

### CODE REVIEW

**Comment by pik694** [comment]
Refers #80

File: contracts/RootChain.sol [hunk 1]
```diff
@@ -845,29 +847,46 @@ contract RootChain {
 
     /**
      * @dev Given UTXO's position, returns its exit ID.
```
Comment (paulperegud) [inline contracts/RootChain.sol]: This line is misleading - UTXO position is only one of two args.

**Comment by paulperegud** [comment]
Hmm, one more remark. We should detect if tx is an deposit in consistent way. When determining exit priority, we check if `blknum rem 1000 != 0`.

**Comment by pik694** [comment]
Closes #80

**Pull request merged**